compiler/src/dmd/dsymbolsem.d

```diff
@@ -1737,7 +1737,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                     const name = (cast(const(char)[])se.peekData()).xarraydup;
                     uint cnt = setMangleOverride(s, name);
                     if (cnt > 1)
-                        pd.error("can only apply to a single declaration");
```

Comment:
This is not an expression but a declaration, and declaration.error also prints the pragma declaration here

Reply:
fixed